10_disable_cache.dpatch by Frederic Peters <fpeters@debian.org>

Do not cache downloaded files in memory to plug a giant memory
hole (see Debian bug #372166)
--- a/libgphoto2/gphoto2-filesys.c
+++ b/libgphoto2/gphoto2-filesys.c
@@ -1628,9 +1628,6 @@ gp_filesystem_get_file_impl (CameraFiles
 	CR (gp_file_set_type (file, type));
 	CR (gp_file_set_name (file, filename));
 
-	/* Cache this file */
-	CR (gp_filesystem_set_file_noop (fs, folder, file, context));
-
 	/*
 	 * Often, thumbnails are of a different mime type than the normal
 	 * picture. In this case, we should rename the file.
